css/index.css

```diff
@@ -1,3 +1,255 @@
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+  width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+}
+
+@media(max-width: 800px) {
+  .container {
+      width: 100%;
+      background: pink;
+  }
+}
+
+@media(max-width: 500px) {
+  .container {
+      width: 100%;
+      background: gray; 
+  }
+}
+
+
+
+/* header */
+
+header {
+    width: 800px;
+    display: flex;
+    justify-content: flex-end; 
+ }
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  width: 691px;
+  vertical-align: top;
+  margin-top: 35px;
+  position: relative;
+  right: 50px;
+  top: 8px;
+}
+
+header nav a {
+
+  margin-right: 65px;
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+.logo {
+  display: flex;
+  position: relative;
+  left: 765px;
+  top: 12px;
+}
+
+/* cta */
+
+.cta .cta-text {
+  display: inline-block;
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: flex;
+  position: relative;
+  left: 570px;
+  bottom: 400px;
+  
+}
+
+
+.main-content {
+    margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  padding: 0 10px;
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  bottom: 390px;
+  border-top: 1px solid black;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  display: inline-block;
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+  position: relative;
+  display: flex;
+  left: 75px;
+  bottom: 400px;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+  display: flex;
+  position: relative;
+  justify-content: center;
+  bottom: 405px;
+  left: 4px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  display: inline-block;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+  display: flex;
+  position: relative;
+  bottom: 420px;
+  left: 60px; 
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+p.copyright {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  bottom: 360px;
+  left: 20px;
+}
+
+
+
+footer {
+  margin: 32px 0;
```

Comment:
Continue to comb through here and changing the left and right influence properties to responsive fluid units, like percents, rems, etc... a property is left/right influenced if it acts along that horizontal dimension .. so width runs left<---> right so it is a good candidate for a responsive unit, padding - right or margin-left same thing, but height or margin-top which has its action occur top to bottom is not considered since we are changing the size of the viewport which is the size that runs across the horizontal (left to right line) of the screen/browser window